system/containers.toit

```diff
@@ -256,9 +256,10 @@ class ContainerManager extends ContainerServiceDefinition implements SystemMessa
       if allocation.type != FLASH_ALLOCATION_PROGRAM_TYPE: continue.do
       add_flash_image allocation
 
-    // Run through the bundled images in the VM.
+    // Run through the bundled images in the VM, but skip the
+    // first one which is always the system image.
     bundled := container_bundled_images_
-    for i := 0; i < bundled.size; i += 2:
+    for i := 2; i < bundled.size; i += 2:
```

Comment:
Add a comment explaining the `2`.
Each bundled entry seems to take two slots. Explain why.

Reply:
Done.